README.md

```diff
@@ -7,17 +7,27 @@ Gradle Kotlin DSL
 
 Welcome! The _Gradle Kotlin DSL_ provides support for writing [Gradle](http://gradle.org) build scripts using JetBrains' [Kotlin](http://kotlinlang.org) language. It aims to provide Gradle users with a rich, flexible and statically-typed approach to developing build logic in conjunction with the best IDE and tooling experience possible.
 
-You can read more about the project in our [announcement blog post](http://gradle.org/blog/kotlin-meets-gradle) and check out the [frequently asked questions](https://github.com/gradle/kotlin-dsl/wiki/Frequently-Asked-Questions) in the project wiki.
-
 
 Getting Started
 ---------------
 
-The fastest way to get up and running with a Kotlin-based Gradle build is to work with one of the [samples](samples). You'll find complete instructions in the README there.
+The fastest way to get up and running with a Kotlin-based Gradle build is to use [gradle init](https://docs.gradle.org/current/userguide/build_init_plugin.html)
+
+```
+gradle init --dsl kotlin
```

Comment:
Very, very big 👍 on this one. Just putting this in the readme is incredibly valuable.